### PR TITLE
code format in status_manager.go

### PR DESCRIPTION
**Pull request by jinzhejz**
**What this PR does / why we need it**:
code format

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #N/A

**Release note**:
```release-note
None
```

### CODE REVIEW

**Comment by xiangpengzhao** [comment]
please fix the verification failure:

```
I0823 04:22:04.705] Verifying hack/make-rules/../../hack/verify-gofmt.sh
I0823 04:22:16.896] diff ./pkg/kubelet/status/status_manager.go gofmt/./pkg/kubelet/status/status_manager.go
I0823 04:22:16.896] --- /tmp/gofmt964297618	2017-08-23 04:22:15.775462231 +0000
I0823 04:22:16.896] +++ /tmp/gofmt717048265	2017-08-23 04:22:15.775462231 +0000
I0823 04:22:16.896] @@ -23,8 +23,6 @@
I0823 04:22:16.896]  
I0823 04:22:16.897]  	"github.com/golang/glog"
I0823 04:22:16.897]  
I0823 04:22:16.897] -	clientset "k8s.io/client-go/kubernetes"
I0823 04:22:16.897] -	"k8s.io/client-go/kubernetes/scheme"
I0823 04:22:16.897]  	"k8s.io/api/core/v1"
I0823 04:22:16.897]  	apiequality "k8s.io/apimachinery/pkg/api/equality"
I0823 04:22:16.897]  	"k8s.io/apimachinery/pkg/api/errors"
I0823 04:22:16.898] @@ -32,6 +30,8 @@
I0823 04:22:16.898]  	"k8s.io/apimachinery/pkg/types"
I0823 04:22:16.898]  	"k8s.io/apimachinery/pkg/util/diff"
I0823 04:22:16.898]  	"k8s.io/apimachinery/pkg/util/wait"
I0823 04:22:16.898] +	clientset "k8s.io/client-go/kubernetes"
I0823 04:22:16.898] +	"k8s.io/client-go/kubernetes/scheme"
I0823 04:22:16.898]  	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
I0823 04:22:16.899]  	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
I0823 04:22:16.899]  	kubepod "k8s.io/kubernetes/pkg/kubelet/pod"
I0823 04:22:16.899] FAILED   hack/make-rules/../../hack/verify-gofmt.sh	12s
```

File: pkg/kubelet/status/status_manager.go [hunk 1]
```diff
@@ -21,10 +21,10 @@ import (
 	"sync"
 	"time"
 
+	"github.com/golang/glog"
+
 	clientset "k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
```
Comment (xiangpengzhao) [inline pkg/kubelet/status/status_manager.go]: move the two lines after `k8s.io/apimachinery/pkg/util/wait` (L34)

**Review by xiangpengzhao — CHANGES_REQUESTED** [review]

File: pkg/kubelet/status/status_manager.go (hunk 1 above)
Comment (jinzhejz) [inline pkg/kubelet/status/status_manager.go]: the commit is updated, thanks

**Comment by jinzhejz** [comment]
commit is updated

**Pull request closed without merging**